sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/SynchronousMetricStorage.java

```diff
@@ -55,23 +68,20 @@
                 metricDescriptor,
                 () -> sampler.createReservoir(view.getAggregation()));
     return new SynchronousMetricStorage<>(
-        metricDescriptor,
-        aggregator,
-        new InstrumentProcessor<>(aggregator, startEpochNanos),
-        view.getAttributesProcessor());
+        metricDescriptor, aggregator, view.getAttributesProcessor());
```

Comment:
`startEpochNanos` parameter is no longer used, and can be removed.

Reply:
Removed